cmake/Dependencies.cmake

```diff
@@ -93,6 +93,26 @@ if(USE_CUDA)
   endif()
 endif()
 
+# ---[ XPU
+if(USE_XPU)
+  set(CAFFE2_USE_XPU ${USE_XPU})
+  include(${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake)
+  if(NOT CAFFE2_USE_XPU)
```

Comment:
The `CAFFE2_USE_XPU` should always be `TRUE`, right?

Reply:
no, in xpu.cmake we can find the sycl libaray. if no found. set `CAFFE2_USE_XPU` to OFF.